addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1038,8 +1040,11 @@ export class Rtc extends Record {
         browser.clearTimeout(this.sfuTimeout);
         this.sfuClient = undefined;
         this.network = undefined;
+        this.audioContext?.close();
```

Comment:
When an `AudioContext` is closed, it will throw an error (`e.name === "InvalidStateError"`) if it's already closed, you could check the state before, or try/catch or set it as undefined when it's closed.

If you don't set it as undefined when it's closed, leaving a call and entering one again will lead to a traceback whenever it's reached (as we would call `close()` on the closed audioContext of the previous call).

also, maybe a bit of nitpicking but I would declare the audioContext and document it in the class property declarations